src/util/fetchData.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+export const login = async ({ email, password }) => {
+    try {
+        const response = await axios.post(`http://localhost:8000/api/v1/login`, {
```

Comment:
You're going to be using this url a lot. Better to save it in a constant that can be accessed across the whole file.